src/app/controllers/surelia.js

```diff
@@ -274,6 +275,11 @@ var Surelia = function ($scope, $rootScope, $state, $window, $stateParams, local
       return results;
     }
     var suggest_email_delimited = function(term) {
+      if (term.replace(/ /g,'')[term.length - 1] === ',') {
+        return self.nextSuggestion = false;
+      } else {
+        self.nextSuggestion = true;
+      }
       if (self.enableAutocomplete) {
```

Comment:
Only if the last key stroke is equal to ``,``, otherwise, it would reach there. Email suggestion still works.